source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx

```diff
@@ -25,8 +39,7 @@ const SinglePropertyMarker: React.FC<React.PropsWithChildren<SinglePropertyMarke
 
   const getIcon = () => {
     if (isOwned) {
```

Comment:
@FuriousLlama should this if be changed too?

Reply:
This one is fine given the marker will only change if its not owned.